src/state-judicial-district-map/StateJudicialDistrictMap.js

```diff
@@ -0,0 +1,113 @@
+import PropTypes from "prop-types";
```

Comment:
All of this State Judicial District Map code was duplicated verbatim in `VizParolePopulation` and `VizSentencePopulation`.  It now lives in one place here.